okapi-core/src/main/resources/log4j2.properties

```diff
@@ -1,19 +1,25 @@
 status = error
 name = PropertiesConfig
-packages = org.folio.okapi.common.logging
 
 filters = threshold
 
 filter.threshold.type = ThresholdFilter
 filter.threshold.level = info
 
-appenders = console
+appender.full.type = Console
+appender.full.name = FULL
+appender.full.layout.type = PatternLayout
+appender.full.layout.pattern = %d{HH:mm:ss} [${map:requestid}] [${map:tenantid}] [${map:userid}] [${map:moduleid}] %-5p %-20.20C{1} %m%n
```

Comment:
Maybe instead of `%d{HH:mm:ss}` we should do `%d{DEFAULT} ` which includes the date and microseconds like this `2012-11-02 14:34:02,123`. There are other possibilities: https://logging.apache.org/log4j/2.x/manual/layouts.html. I think in a lot of envs (like the rancher ones) this file is just taken as is and having the additional info could help debug.

It also might help self-hosters who also may just take this file without much thought and then be surprised that they can't see dates in their logs.

Reply:
I'm ok with that.. @skoczko ? @julianladisch ?

Reply:
The PatternLayout should be consistent across all FOLIO modules.
Currently we always use `%d{HH:mm:ss}`: https://github.com/search?l=Java+Properties&q=org%3Afolio-org+PatternLayout&type=Code
When we change this default it should be coordinated and changed in all modules, and planned for a specific flower release, similar to https://wiki.folio.org/display/TC/Officially+Supported+Technologies
Therefore the PatternLayout change should not be included in this PR.
Please create a FOLIO Jira and discuss it in the #sys-ops Slack channel.
Inplementers can always provide a log4j2.properties file that overwrites the FOLIO default.

Reply:
I guess I would ask why the pattern layout should be consistent between okapi and the modules. I do agree that it breaks the symmetry with the modules but would there be any practical downside in terms of how the logs are used?

I feel like it's an oversight that we're not putting a date in there.

I'll create a JIRA as you suggest and post something on sys ops in slack to get a conversation going on it. In any case I don't think merging what Adam has should be held up by this.

Reply:
Create a JIRA please.